<launch>
    <arg name="lane_angle_error_topic" value="/lane_detection/angle" >
    <arg name="order" value="0">
    <arg name="linear_speed" value="0.5">
    <arg name="angular_speed" value="0.1">

    <!-- If you want to launch the node multiple time, change the name tag's value -->
    <node name="lane_following_node" pkg="turtlebot3_lane_following" type="lane_following_node" output="screen">
        <param name="angle_order" value="$(arg order)"/>
        <param name="robot_linear_speed" value="$(linear_speed)"/>
        <param name="robot_angular_speed" value="$(angular_speed)"/>

        <!-- Modify parameters depending on your system -->
        <param name="propotionnal_constant" value="0.5"/>
        <param name="integration_constant" value="0.5"/>
        <param name="derivation_constant" value="0.5"/>

        <param name="sensor_topic" value="$(arg lane_angle_error_topic)"/>
    </node>
</launch>
